saml_reader/saml/parser.py

```diff
@@ -171,6 +171,12 @@ def from_base64(cls, base64, url_decode=False):
             (BaseSamlParser) parsed SAML response object
         """
         value = base64 if not url_decode else unquote(base64)
+        # Strip whitespace
+        pattern = re.compile(r'\s+')
+        value = re.sub(pattern, '', value)
```

Comment:
Since we don't reuse the regex, there's no need to compile the regex first. So go ahead and just replace `pattern` with the raw string.

(I copy-pasted that code from StackExchange)

Reply:
done